sport/test/football/feed/CompetitionsTest.scala

```diff
@@ -0,0 +1,80 @@
+package football.feed
+
+import java.time.{Clock, Duration, ZoneId, ZonedDateTime}
+
+import feed.Competitions
+import org.scalatest._
+import pa.FootballMatch
+import test.FootballTestData.{competitions, result, liveMatch, fixture}
+
+class CompetitionsTest extends FreeSpec with Matchers with OptionValues {
+  "Competitions" - {
+    Seq(
+      (false, Duration.ofSeconds(1), true),
+      (false, Duration.ofMinutes(1), true),
+      (false, Duration.ofMinutes(14).plusSeconds(59), true),
+      (false, Duration.ofMinutes(15), false),
+      (false, Duration.ofMinutes(16), false),
+      (true, Duration.ofMinutes(16), true),
+      (true, Duration.ofMinutes(20), true),
+    ) foreach { testcase =>
+      (s"isMatchLiveOrAboutToStart returns ${testcase._3} if there's a match that started ${testcase._2} ago") in {
+        val matches: Seq[FootballMatch] =
+          matchesWithLiveMatchAtCurrentMinusDuration(testcase._2, testcase._1)
+
+        val testCompetition = competitions(1).copy(matches = matches)
+        val competitionsList = Competitions(Seq(testCompetition))
+
+        val isMatchLiveOrAboutToStart = competitionsList.isMatchLiveOrAboutToStart(competitionsList.matches, clock)
+
+        isMatchLiveOrAboutToStart should equal(testcase._3)
+      }
+    }
+
+    Seq(
+      (false, Duration.ofSeconds(1), true),
+      (false, Duration.ofMinutes(1), true),
+      (false, Duration.ofMinutes(4).plusSeconds(59), true),
+      (false, Duration.ofMinutes(5), false),
+      (false, Duration.ofMinutes(10), false),
+      (true, Duration.ofMinutes(5), true),
+      (true, Duration.ofMinutes(10), true),
+    ) foreach { testcase =>
+      (s"isMatchLiveOrAboutToStart returns ${testcase._3} if there's a match with liveMatch status ${testcase._1} that will start in ${testcase._2}") in {
+        val matches: Seq[FootballMatch] =
+          matchesWithLiveMatchAtCurrentPlusDuration(testcase._2, testcase._1)
+        val testCompetition = competitions(1).copy(matches = matches)
+        val competitionsList = Competitions(Seq(testCompetition))
+        val isMatchLiveOrAboutToStart = competitionsList.isMatchLiveOrAboutToStart(competitionsList.matches, clock)
+
+        isMatchLiveOrAboutToStart should equal(testcase._3)
+      }
+    }
+  }
+
+  private val zone = ZoneId.of("Europe/London")
+
+  private val today = ZonedDateTime.now().withZoneSameInstant(zone)
+
+  private val clock = {
+    val fixedDate = today.toInstant
+    Clock.fixed(fixedDate, zone)
+  }
+
+  def matchesWithLiveMatchAtCurrentMinusDuration(duration: Duration, liveMatchStatus: Boolean = false) = {
```

Comment:
I'd consider inlining these functions so the test data is all in one place. As it stands, the reader has to jump around the file a bit, and each of these are only used in one place anyway. Only a little thing, no dramas!